Bikes/Bikes.Infrastructure.InMemory/Repositories/InMemoryBikeModelRepository.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Bikes.Domain.Models;
+using Bikes.Domain.Repositories;
+using Bikes.Infrastructure.InMemory.Seeders;
+
+namespace Bikes.Infrastructure.InMemory.Repositories;
+
+/// <summary>
+/// Implementing a repository for the BikeModel class 
+/// </summary>
+public class InMemoryBikeModelRepository : IRepository<BikeModel, int>
+{
+    private readonly List<BikeModel> _items = [];
```

Comment:
Можно не инициализировать, он у вас в конструкторе инициализируется.

```suggestion
    private readonly List<BikeModel> _items;
```